### ISSUE TITLE
Windows doesn't have lib in front of libraries.

### ISSUE THREAD

**Issue by fire** [GuilhermeGSousa/godot-motion-matching#81]
```
windows.debug.x86_32 = "bin/windows/gdmotionmatching.windows.template_debug.x86_32.dll"
windows.release.x86_32 = "bin/windows/gdmotionmatching.windows.template_release.x86_32.dll"
windows.debug.x86_64 = "bin/windows/gdmotionmatching.windows.template_debug.x86_64.dll"
windows.release.x86_64 = "bin/windows/gdmotionmatching.windows.template_release.x86_64.dll"
```

**Comment by fire** [comment]
I don't think the the demo is up to date.

**Comment by GuilhermeGSousa** [comment]
Fixed in #82
